src/main/java/com/conveyal/datatools/manager/models/Project.java

```diff
@@ -126,6 +126,14 @@ public Collection<Label> retrieveProjectLabels(boolean isAdmin) {
     @BsonIgnore
     public Collection<Label> labels;
 
+    @JsonProperty
+    public int feedSourceCount() {
+        if (retrieveProjectFeedSources() == null) {
+            return -1;
+        }
+        return retrieveProjectFeedSources().size();
```

Comment:
Perhaps try `Persistence.feedSources.count(eq("projectId", this.id));` instead. That way you'll get just the count and not all the objects.

Reply:
Much better thank you so much!